src/prelude/internals.mo

```diff
@@ -411,3 +411,17 @@ func @create_actor_helper(wasm_module_ : Blob, arg_ : Blob) : async Principal =
 func @call_raw(p : Principal, m : Text, a : Blob) : async Blob {
   await (prim "call_raw" : (Principal, Text, Blob) -> async Blob) (p, m, a);
 };
+
+// stable memory footprint
+func @query_stable_var_footprint(self : actor {}, method : ?Text) : async Nat64 {
```

Comment:
Why not just fix the method name to some well known-name like `__stable_variable_footprint` - I guess you still need to use the dynamic call because you can't use express this method name as a valid identifier - we should really add some way of escaping identifiers so we can interact with foreign canisters with funny names.

Reply:
I was thinking of adding another primitive that gets the `?<footpring-query-name>`, and I need a switch anyway to only create it when necessary (the user wants it).

Reply:
I'd just always create it, or just create it on demand if the code requires it - asking the user to set a flag for this seems awkward,  just to save a few bytes in the image. It's not that many, right?